adm_program/system/function.php

```diff
@@ -697,6 +697,48 @@ function admFuncGetDirectoryEntries($directory, $searchType = 'file')
 }
 
 /**
+ * Gets the total, free and used disk space.
+ * @param string $path Path of the filesystem
+ * @return array<string,int>
```

Comment:
Please add a description here how the array will be filled with each value. Maybe additional an example.

Reply:
It's a short function and you may see it in the code but you should also have in mind that we can create a documentation without the code and than you don't know how the array looks like.